modules/react/select/lib/Select.tsx

```diff
@@ -25,27 +26,33 @@ export interface SelectInputProps extends ExtractProps<typeof TextInput>, CSProp
   inputStartIcon?: CanvasSystemIcon;
 }
 
-const selectInputStyles = createStyles({
-  caretColor: 'transparent',
-  cursor: 'default',
-  '&::selection': {
-    backgroundColor: 'transparent',
+const selectInputStencil = createStencil({
+  base: {
+    caretColor: 'transparent',
+    cursor: 'default',
+    '&::selection': {
+      backgroundColor: 'transparent',
+    },
   },
 });
 
-const iconStyles = createStyles({
-  position: 'absolute',
-  pointerEvents: 'none',
+const selectIconsStencil = createStencil({
+  base: {
+    position: 'absolute',
+    pointerEvents: 'none',
+  },
 });
-const hiddenSelectInputStyles = createStyles({
-  position: 'absolute',
-  top: 0,
-  bottom: 0,
-  left: 0,
-  right: 0,
-  opacity: 0,
-  cursor: 'default',
-  pointerEvents: 'none',
+const hiddenSelectInputStencil = createStencil({
+  base: {
+    position: 'absolute',
+    top: system.space.zero,
+    bottom: system.space.zero,
+    left: system.space.zero,
+    right: system.space.zero,
+    opacity: system.space.zero,
```

Comment:
`system.space.zero` is rem value, opacity is not px/rem value, it's just a number